lib/bike_index/requester.rb

```diff
@@ -0,0 +1,40 @@
+require './lib/bike_index' # why doesn't this get linked up automagically?
+
+class BikeIndex::Requester
+  include HTTParty
+  base_uri 'https://bikeindex.org/api/v3'
+
+  def initialize(user=nil)
+    @user = user if user.present?
+  end
+
+  def get_bikes
+    get_request
+  end
+
+  def get_request
+    # There were two ways of doing this, in regards to the access_token.
+    # This is the only one that worked but didn't see ideal? I can't remember why.
```

Comment:
I think you'll be fine here. It would be better to use basic auth, or something that didn't put the access_token in the url, but I don't think it's devastating